src/gui/server_widget.py

```
@@ -8,12 +8,15 @@
 import requests
 import os
 
+from ..printer.manager import PrinterManager
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)